### PR TITLE
Utility script to load id and parent reference tags

### PR DESCRIPTION
**Pull request by schandrika**
1. Added Utility script to load id and parent reference tags for topics that have naming convention as /campus/building/device/<optional sub device>/point

2. Removed unused test script

---
This change is [Reviewable](https://reviewable.io/reviews/volttron/volttron/1466)

### CODE REVIEW

File: scripts/tagging_scripts/insert_id_and_ref_tags.py [hunk 1]
```diff
@@ -0,0 +1,256 @@
+"""
+Utility script to read the list of topics from mongodb and insert mandatory
+id and reference tags for these topics. The script only works for topics
+with the naming convention /campus/building/device/<optional sub device>/point.
+A error will be displayed for any topics that do not conform to this naming
```
Comment (craig8) [inline scripts/tagging_scripts/insert_id_and_ref_tags.py]: Are you sure its /hcampus generally we don't have a rooted path...

File: scripts/tagging_scripts/insert_id_and_ref_tags.py [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+import re
+import sqlite3
+from pymongo import MongoClient
+from pymongo.errors import BulkWriteError
+import datetime
+
+## User configuration - start ##
+
+#Connection string to the mongo db from which list of topics should be read
+topics_connection_string = "mongodb://reader:volttronReader@vc-db.pnl.gov" \
```
Comment (craig8) [inline scripts/tagging_scripts/insert_id_and_ref_tags.py]: Pleae stop puting username/passwords in the files.  Use a config file or argparse pass as a parameter!

File: scripts/tagging_scripts/insert_id_and_ref_tags.py [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+                           ":27017" \
+                           "/prod_historian"
+topic_table = "topics"
+# Regular expression that matches all device topics that conform to the
+# naming pattern /campus/building/device/<optional sub device>/point
+device_topics = "^PNNL"
```
Comment (craig8) [inline scripts/tagging_scripts/insert_id_and_ref_tags.py]: /campus ?

File: scripts/tagging_scripts/insert_id_and_ref_tags.py [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+    write_to_mongo = True
+
+write_to_sqlite = False
+if tags_db_sqlite:
+    sqlite_connection = sqlite3.connect(
+        tags_db_sqlite,
+        detect_types = sqlite3.PARSE_DECLTYPES | sqlite3.PARSE_COLNAMES
+    )
+    sqlite_connection.execute("PRAGMA CACHE_SIZE=-2000")
+
```
Comment (craig8) [inline scripts/tagging_scripts/insert_id_and_ref_tags.py]: Why a - cache size?

Comment (schandrika) [reply scripts/tagging_scripts/insert_id_and_ref_tags.py]: without a large enough cache sqlite queries can be super slow.

File: scripts/tagging_scripts/insert_id_and_ref_tags.py (hunk 2 above)
Comment (schandrika) [reply scripts/tagging_scripts/insert_id_and_ref_tags.py]: sorry. removed

File: scripts/tagging_scripts/insert_id_and_ref_tags.py (hunk 3 above)
Comment (schandrika) [reply scripts/tagging_scripts/insert_id_and_ref_tags.py]: This is a documentation issue.  I fixed it

**Comment by schandrika** [comment]
Review status: 0 of 2 files reviewed at latest revision, 4 unresolved discussions.

---

*[scripts/tagging_scripts/insert_id_and_ref_tags.py, line 5 at r1](https://reviewable.io:443/reviews/volttron/volttron/1466#-Ksz_uOfiQf0p5kKazDe-r1-5:-Ktrlah9K7BW8DG1RW65:bxa2a1n) ([raw file](https://github.com/volttron/volttron/blob/f8edd42d33bfe45af63c5fe5a456c1940b92cc3b/scripts/tagging_scripts/insert_id_and_ref_tags.py#L5)):*
<details><summary><i>Previously, craig8 (Craig) wrote…</i></summary><blockquote>

Are you sure its /hcampus generally we don't have a rooted path...
</blockquote></details>

fixed documentation. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/volttron/volttron/1466)*

File: scripts/tagging_scripts/insert_id_and_ref_tags.py (hunk 4 above)
Comment (craig8) [reply scripts/tagging_scripts/insert_id_and_ref_tags.py]: Yes but why is it negative!

Comment (schandrika) [reply scripts/tagging_scripts/insert_id_and_ref_tags.py]: it indicates it is in kilobytes and not pages.

Comment (craig8) [reply scripts/tagging_scripts/insert_id_and_ref_tags.py]: Ahh...that would be a good spot for a comment...as I had no clue.

**Comment by craig8** [comment]
Reviewed 1 of 2 files at r1, 1 of 1 files at r2, 2 of 2 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/volttron/volttron/1466)*

**Comment by craig8** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[scripts/tagging_scripts/insert_id_and_ref_tags.py, line 64 at r1](https://reviewable.io:443/reviews/volttron/volttron/1466#-Ksz_uOfiQf0p5kKazDe-r1-64:-Ktvu--rHTH5ARjuk81b:bb6ywmx) ([raw file](https://github.com/volttron/volttron/blob/f8edd42d33bfe45af63c5fe5a456c1940b92cc3b/scripts/tagging_scripts/insert_id_and_ref_tags.py#L64)):*
<details><summary><i>Previously, craig8 (Craig) wrote…</i></summary><blockquote>

Ahh...that would be a good spot for a comment...as I had no clue....of course goole would have told me it to probably.
</blockquote></details>

LGTM

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/volttron/volttron/1466)*

**Pull request merged**